events/wc_culture_conversion_events.txt

```diff
@@ -0,0 +1,185 @@
+# ###########################################
+
+namespace = WCMLT
+
+# ###########################################
+# # Province conversions                    #
+# ###########################################
+
+# Human province in Barrens/Dustwallow converts to Theramorean
+province_event = {
+	id = WCMLT.1
+	desc = EVTDESC_WCMLT_1
+	picture = GFX_evt_lordaeron_coast
+
+	trigger = {
+		should_evolve_cultures_trigger = yes
+		year = 600
+		culture_group = human_group
+		colonial_culture_trigger = no
+		OR = {
+			region = world_kalimdor_barrens		 
+			region = world_kalimdor_dustwallow
+			region = island_theramore
+			region = island_alcaz
+		}
+		any_province_holding = {
+			NOT = { holding_type = nomad }
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 450
+		modifier = {
+			factor = 0.3
+			OR = {
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots
+				}
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots_and_slower_normal
+				}
+			}
+		}
+	}
+
+	option = {
+		name = EVTOPTA_WCMLT_1
+		culture = theramore
+	}
+}
+
+# Orcish province in Kalimdor converts to Durotaric
+province_event = {
+	id = WCMLT.2
+	desc = EVTDESC_WCMLT_2
+	picture = GFX_evt_durotar
+
+	trigger = {
+		should_evolve_cultures_trigger = yes	
+		year = 600
+		culture_group = orc_group
+		colonial_culture_trigger = no
+		region = world_kalimdor_durotar
+		any_province_holding = {
+			NOT = { holding_type = nomad }
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 450
+		modifier = {
+			factor = 0.3
+			OR = {
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots
+				}
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots_and_slower_normal
+				}
+			}
+		}
+	}
+
+	option = {
+		name = EVTOPTA_WCMLT_2
+		culture = durotaric
+	}
+}
+
+# Human province in Tanaris/Uldum converts to Wastewander
+province_event = {
+	id = WCMLT.3
+	desc = EVTDESC_WCMLT_3
+	picture = GFX_evt_sand
+
+	trigger = {
+		should_evolve_cultures_trigger = yes		
+		year = 600
+		culture_group = human_group
+		colonial_culture_trigger = no
+		OR = {
+			region = world_kalimdor_tanaris		 
+			region = world_kalimdor_uldum
+		}
+		any_province_holding = {
+			NOT = { holding_type = nomad }
+		}
+	}
+	
+	mean_time_to_happen = {
+		months = 450
+		modifier = {
+			factor = 0.3
+			OR = {
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots
+				}
+				has_game_rule = {
+					name = culture_conversion
+					value = faster_melting_pots_and_slower_normal
+				}
+			}
+		}
+	}
+
+	option = {
+		name = EVTOPTA_WCMLT_3
+		culture = wastewander
+	}
+}
+
+# Human province in Kalimdor converts to Western
+province_event = {
+	id = WCMLT.4
+	desc = EVTDESC_WCMLT_4
+	picture = GFX_evt_forest
+
+	trigger = {
+		should_evolve_cultures_trigger = yes		
+		year = 600
+		culture_group = human_group
+		region = world_kalimdor
```

Comment:
Durotar and Kalimdor isles is not a parts of Kalimdor region, so human cultures in Durotar and Kalimdor isles will not be converted into new culture

<details>
<summary>Click to expand</summary>

![ck2_10](https://user-images.githubusercontent.com/46576860/56255206-aaffda80-60cc-11e9-9bcb-c8a63840b0eb.png)

</details>